### PR TITLE
Remove bogus buildable actors from TS.

### PR DESCRIPTION
**Pull request by pchote**
This does an initial rough polish pass over the TS tech tree:
- Remove Buildable trait from special and civilian actors, which are otherwise buildable (even without cheats).
- Fix the mutant hijacker.
- Fix the EMP generator icon.

### CODE REVIEW

File: mods/ts/rules/infantry.yaml [hunk 1]
```diff
@@ -268,15 +257,10 @@ JUMPJET:
...
 	Inherits: ^Infantry
-	Buildable:
-		Queue: Infantry
-		BuildPaletteOrder: 60
-		Prerequisites: ~nahand, natech
 	Valued:
 		Cost: 700
 	DisguiseToolTip:
 		Name: Chameleon Spy
-		Description: Infiltrates enemy structures to gather \nintelligence. Exact effect depends on the \nbuilding infiltrated.\n  Strong vs Nothing\n  Weak vs Everything\n  Special Ability: Disguised
```
Comment (Mailaender) [inline mods/ts/rules/infantry.yaml]: How are these removals any helpful?

File: mods/ts/rules/vehicles.yaml [hunk 2]
```diff
@@ -837,4 +628,3 @@ STNK:
 	RenderVoxels:
 	WithVoxelBody:
 	-MustBeDestroyed:
-
```
Comment (Mailaender) [inline mods/ts/rules/vehicles.yaml]: This will probably be re-added by the upgrade scripts.

File: mods/ts/sequences/infantry.yaml [hunk 3]
```diff
@@ -530,7 +529,6 @@ umagon:
 	die6: electro
 		Length: *
 		Tick: 80
-	icon: umagicon
```
Comment (Mailaender) [inline mods/ts/sequences/infantry.yaml]: Why are you removing this?!

File: mods/ts/sequences/infantry.yaml [hunk 4]
```diff
@@ -848,7 +846,6 @@ chamspy:
 	die6: electro
 		Length: *
 		Tick: 80
-	icon: chamicon
```
Comment (Mailaender) [inline mods/ts/sequences/infantry.yaml]: Pretty sure this icon is correct.

File: mods/ts/sequences/vehicles.yaml [hunk 5]
```diff
@@ -104,7 +85,6 @@ gghunt:
 		Facings: 1
 		Length: 8
 		ShadowStart: 8
-	icon: seekicon
```
Comment (Mailaender) [inline mods/ts/sequences/vehicles.yaml]: Why are you removing this?

File: mods/ts/sequences/infantry.yaml (hunk 4 above)
Comment (pchote) [inline mods/ts/sequences/infantry.yaml]: In the original game chamspy is not buildable and the icon is instead used by ghost stalker.

File: mods/ts/sequences/infantry.yaml (hunk 3 above)
Comment (pchote) [inline mods/ts/sequences/infantry.yaml]: Umagon isn't buildable, so it isn't needed.

File: mods/ts/sequences/vehicles.yaml (hunk 5 above)
Comment (reaperrr) [inline mods/ts/sequences/vehicles.yaml]: the hunter-seeker should be a super-power and not buildable, so at least this removal looks right to me.

File: mods/ts/rules/infantry.yaml (hunk 1 above)
Comment (pchote) [inline mods/ts/rules/infantry.yaml]: They aren't used (since they aren't buildable), and some of them are completely wrong.

File: mods/ts/sequences/infantry.yaml (hunk 3 above)
Comment (Mailaender) [inline mods/ts/sequences/infantry.yaml]: That is not a valid point. It is there and has to be assigned somewhere so we keep track of it.

File: mods/ts/sequences/infantry.yaml (hunk 4 above)
Comment (reaperrr) [inline mods/ts/sequences/infantry.yaml]: Wasn't buildable in the original and the icon is already used by the buildable hijacker (in the original, too), so this removal is correct.

File: mods/ts/sequences/infantry.yaml (hunk 3 above)
Comment (reaperrr) [inline mods/ts/sequences/infantry.yaml]: Umagon was not buildable in the original, but I agree that removing this is unnecessary.

File: mods/ts/rules/vehicles.yaml (hunk 2 above)
Comment (reaperrr) [inline mods/ts/rules/vehicles.yaml]: Agreed.

File: mods/ts/rules/infantry.yaml (hunk 1 above)
Comment (reaperrr) [inline mods/ts/rules/infantry.yaml]: I guess pchote removed this since the spy isn't buildable, but I agree that there would be no harm in keeping this, saves us the hassle of adding it back if we ever decide to make the spy buildable.

Comment (reaperrr) [inline mods/ts/rules/infantry.yaml]: Sorry, took me too long to answer, had not refreshed the page.

File: mods/ts/rules/vehicles.yaml (hunk 2 above)
Comment (pchote) [inline mods/ts/rules/vehicles.yaml]: There were actually two newlines at the end of this file.  This removal brings us back to the standard one.

Comment (reaperrr) [inline mods/ts/rules/vehicles.yaml]: That's the danger of not looking at the full changes, I guess... nevermind, then.

**Comment by reaperrr** [comment]
Looks good to me :+1: 
I can see why @Mailaender would like to keep the icon sequences and descriptions, but personally I don't think the removals are blockers since those actors are indeed non-buildable, singleplayer/decoration-only in the original, so until we decide to make some of them buildable in mp or sp, we don't need these icon sequences & descriptions.

**Comment by Mailaender** [comment]
It is still undoing my work of identifying and setting up those sequences for no reason.

File: mods/ts/mod.yaml [hunk 6]
```diff
@@ -64,6 +64,7 @@ Rules:
 	./mods/ts/rules/misc.yaml
 	./mods/ts/rules/ai.yaml
 	./mods/ts/rules/civilian.yaml
+	./mods/ts/rules/civilianvehicles.yaml
```
Comment (Phrohdoh) [inline mods/ts/mod.yaml]: Could you name this `civilian-vehicles` for readability?

**Comment by Phrohdoh** [comment]
Other than my filename change request, :+1: this can be merged.
It is nice to slowly get TS to a presentable state.

**Comment by pchote** [comment]
Restored umagon cameo and renamed `civilian-vehicles.yaml`.

File: mods/ts/sequences/infantry.yaml [hunk 7]
```diff
@@ -300,7 +300,6 @@ weedguy:
 		Length: 3
 		ShadowStart: 376
 		Tick: 1000
-	icon: weaticon
```
Comment (Mailaender) [inline mods/ts/sequences/infantry.yaml]: Why did you remove this icon?

Comment (pchote) [inline mods/ts/sequences/infantry.yaml]: weedguy is not a real actor in TS - it was a hack to make the chemical missile work.  `weaticon` was instead used for the Elite Cadre unit in FS.

**Comment by pchote** [comment]
Reverted the mutant and chamspy icon removals so that we can get this merged as there are now multiple other prs that depend on this.  I still think think this is silly and wasteful, however.

**Comment by Mailaender** [comment]
> Remove Buildable trait from special and civilian actors, which are otherwise buildable (even without cheats).

This was not really helpful. #4496 is not yet merged so now I have to revert this for debugging.
